collection/screenshot/scan-screen.yml

```diff
@@ -0,0 +1,13 @@
+rule:
+  meta:
+    name: scan screen
```

Comment:
What does `scan` refer to? How is this different to `capture screenshot`?

Reply:
I used the word `scan` because of the way this sample uses `BitBlt` to loop through and record segments of the source screen. However, `capture screenshot` is probably a better fit. Should I merge this rule with `capture screenshot`?

Reply:
If this is essentially screenshot capturing I'd say merge. If it's a different mechanism a separate rule with according doc works as well for me.

Reply:
I agree.  There will probably be another rule at somepoint that is dealing with screen recording (using the `capCaptureSequence` or the other multimedia libraries)

Reply:
I'll merge the two rules for now and then merge the PR. Thank you for the responses!